autopilot-daemon/pkg/handlers/handler.go

```diff
@@ -48,19 +40,20 @@ func SystemStatusHandler() http.Handler {
 			klog.Info("Running iperf3 on hosts ", hosts, " or job ", jobName)
 			w.Write([]byte("Running iperf3 on hosts " + hosts + " or job " + jobName + "\n\n"))
 			checks = strings.Trim(checks, "iperf")
-			plane := r.URL.Query().Get("plane")
-			if plane == "" {
-				plane = "data"
-			}
-			sourceNode := r.URL.Query().Get("source")
-			if sourceNode == "" {
-				sourceNode = "None"
+			workload := r.URL.Query().Get("workload")
```

Comment:
In this case it will default to the defaults listed in the python script -- if we set defaults here as well, then that means we'll have multiple places we'll need to change in the future if any of the _defaults_ should be changed.

In the other cases such as individual `client` or `server` calls, it's wrong to assume what the user would intend. For example, what would a reasonable default be for either the individual server or client calls?